Description: Add required feature for dependency dyn
Bug: https://github.com/frozenlib/structmeta/pull/2
Last-Update: 2023-03-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -39,3 +39,4 @@
 
 [dependencies.syn]
 version = "1.0.82"
+features = ["extra-traits"]
